### PR TITLE
Break out Repo access from API.UserController

### PR DESCRIPTION
**Pull request by wojtekmach**
See #386.

### CODE REVIEW

File: web/controllers/api/user_controller.ex [hunk 1]
```diff
@@ -1,26 +1,12 @@
 defmodule HexWeb.API.UserController do
   use HexWeb.Web, :controller
+  alias HexWeb.Users
```
Comment (ericmj) [inline web/controllers/api/user_controller.ex]: Add this alias to `HexWeb.Web`.

Comment (wojtekmach) [inline web/controllers/api/user_controller.ex]: done

**Comment by ericmj** [comment]
Feel free to merge when ready.

**Comment by wojtekmach** [comment]
cool, thanks! the new github review tool and flow looks pretty neat :)

**Pull request merged**